tests/commands/dump.py

```diff
@@ -13,6 +13,7 @@
 
 
 @pytest.mark.cmd
+@pytest.mark.django_db
 def test_dump_noargs():
     """Dump requires an output option."""
     with pytest.raises(CommandError) as e:
```

Comment:
hmmm - on im not sure - im backporting the things that were failing once we remove the db marker from the fixtures - and im pretty sure this was one of them.

i can check